docs/smart/tutorial.md

```diff
@@ -0,0 +1,335 @@
+---
```

Comment:
This file was removed from `master`, since this content is being managed elsewhere (on docs.blockstack repo) -- can you remove this from your PR?